deployments/image/plugin/start-yunikorn-plugin.sh

```diff
@@ -17,13 +17,15 @@
 # limitations under the License.
 
 cd "${HOME}" || exit
+
+# Embed scheduler name into config file
+sed 's/schedulerName:.*/schedulerName: '"${SCHEDULER_NAME}"'/' < "${SCHEDULER_CONFIG}" > /tmp/scheduler-config.yaml || exit
```

Comment:
We need to move this out of this shell script and into pre-processing as this requires `sed` to be installed in the image which might not be the case for stripped down setups.

Reply:
The point was to allow customizing the scheduler name, which as of 1.23 has to be in the scheduler-config file. However, as part of the configv2 redesign, we're removing the ability to change the scheduler name (which never really worked anyway). So I'm removing this code.